source/projects/jungle_beat/iteration_2.md

```diff
@@ -1,27 +1,28 @@
 # Iteration 2 - Append, Prepend and Insert (Multiple Nodes)
 
-Now we have nodes and a `JungleBeat` class that manages the list. Next step is to add the `insert` and `prepend` methods as well as ensuring that `insert`, `append` and `prepend` works for both single and multiple nodes.
+Now we have nodes and a `LinkedList` class that manages the list. Next step is to add the `insert` and `prepend` methods as well as ensuring that `insert`, `append` and `prepend` works for both single and multiple nodes.
 
 `prepend` will add nodes to the beginning of the list.
 
 `insert` will insert one or more elements at a given position in the list. It takes two parameters, the first one is the position at which to insert nodes, the second parameter is the string of data to be inserted.
 
+`prepend` and `append` takes a string parameter with data separated by spaces. Given the string "plop sup dip", three nodes should be added to the list.
+
 Expected behavior:
 
 ```ruby
-> require "./lib/jungle_beat"
-> jb = JungleBeat.new("deep")
-=> 1
-> jb.append("plop sup")
+> require "./lib/linked_list"
+> list = LinkedList.new
+> list.append("plop sup")
```

Comment:
wait i think this style (append string of words actually appends multiple nodes) is only for the JungleBeat wrapper right?